addons/html_editor/static/src/main/list/list_plugin.js

```diff
@@ -835,4 +851,79 @@ export class ListPlugin extends Plugin {
             }
         }
     }
+
+    applyListFormat(formatName, { formatProps } = {}) {
```

Comment:
Unless I'm mistaken, this function does not apply a list format, it applies a format to a list, correct? Therefore I would name it `applyFormatToList` instead.

Reply:
You are correct we apply apply format to a list, particularly list item. How about `applyFormatToListItem` or should i just keep it `applyFormatToList` also the same is true for `applyListColor` shall I change this too?

Reply:
Yes, `applyFormatToListItem` and `applyColorToListItem` sound great, thanks!

Reply:
Done!